private/pkg/uuidutil/uuidutil.go

```diff
@@ -78,3 +78,21 @@ func ValidateDashless(dashless string) error {
 	_, err := FromDashless(dashless)
 	return err
 }
+
+// FromStringSlice returns a slice of uuids from the string slice.
+//
+// This only accepts uuids with dashes.
+func FromStringSlice(s []string) ([]uuid.UUID, error) {
+	var parsedUUIDS []uuid.UUID
+	for _, stringID := range s {
+		if len(stringID) != 36 {
+			return nil, fmt.Errorf("expected uuid to be of length 36 but was %d: %s", len(s), s)
+		}
+		parsed, err := uuid.FromString(stringID)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Should we not just use `FromString` here?

Reply:
yes i was testing u 🙃